pubspec.yaml

```diff
@@ -55,3 +58,8 @@ flutter:
 
 assets:
   - .env.json
+
+global_options:
+  freezed:freezed:
+    runs_before:
```

Comment:
Well done here - we don't use this combination but it's a good idea to write the configuration here for future use.

*But* shouldn't this go into `build.yaml` instead?

Reply:
Definitively